config/image-cache.js

```diff
@@ -32,17 +31,32 @@ const TOKEN_NUMBER = process.argv[4];
 const IMAGE_RESIZE_WIDTH = parseInt(process.env['IMAGE_CACHE_RESIZE_WIDTH'] ?? '300');
 const GCS_BUCKET_NAME = process.env['IMAGE_CACHE_GCS_BUCKET_NAME'];
 const GCS_OBJECT_NAME_PREFIX = process.env['IMAGE_CACHE_GCS_OBJECT_NAME_PREFIX'];
-const GCS_AUTH_TOKEN = process.env['IMAGE_CACHE_GCS_AUTH_TOKEN'];
```

Comment:
It might still be useful to allow a user to pass in a long-lived GCS auth token if they're not hosting the API within a GKE cluster like we are, or for local testing. Our tokens will rotate every so often for increased security, but that won't always be the case for other peoples' setups.